app/_locales/en/messages.json

```diff
@@ -1814,6 +1820,6 @@
     "message": "MetaMask will never ask for your seed phrase!"
   },
   "zeroGasPriceOnSpeedUpError": {
-    "message":"Zero gas price on speed up"
+    "message": "Zero gas price on speed up"
```

Comment:
My auto formatter did this with out me noticing. I've turned it off now. I can go back and fix this is you want though.

Reply:
Strange - I hadn't realized these files weren't covered by `eslint`. This change seems fine to leave in this PR - I'll make a separate PR to adjust our lint script to prevent this going forward.

Reply:
It looks like linting this kinda thing is more difficult than I had anticipated.

We are already using `eslint-plugin-json`, but we're not linting `.json` files which renders this plugin kinda pointless. But after fixing that, it still didn't catch this issue. It only ensures that the `.json` files contain valid JSON, it doesn't enforce normal linting rules like `key-spacing`. We'll just have to catch these manually I guess.

Reply:
I've put up a PR for this here: #6852